openapi/video-openapi.yaml

```diff
@@ -2829,7 +2862,7 @@ externalDocs:
   url: https://getstream.io/video/docs/
 info:
   title: Stream Video API
-  version: v80.4.1
+  version: v80.4.2
```

Comment:
@guerinoni @tbarbugli - this is not a patch version change. Isn't it? Because we are changing the type from int to enum?

Reply:
Right... But this is the version of backend, not the version of API

Reply:
are you sure about the change? i dont think we return a string for this field